services/QuillLMS/db/structure.sql

```diff
@@ -938,7 +938,7 @@ CREATE TABLE public.change_logs (
     action character varying NOT NULL,
```

Comment:
There are some changes that are not included in your PR. Can we ensure that the schema file is consistent?

Reply:
@cissyxyu Good question.  The [diff](https://github.com/empirical-org/Empirical-Core/pull/8027/files#diff-f4525a8efa84d2b12114079828637b4260222ff089ba8488d02cf1890fe332c3R7391) shows [two](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/db/migrate/20210712213724_allow_null_on_change_log_user.rb) [migrations](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/db/migrate/20210722144950_add_highlight_prompt_to_passages.comprehension.rb) that were not checked into `db/structure.sql`